task: Add deprecation message on MongoDB CLI, and remove dependabot automation (#3673)
mongodb/mongodb-atlas-cli: README.md

# MongoDB CLI (Deprecated)

# 🚨 Important Notice
MongoDB CLI has moved to a new repository! You can find it [here](https://github.com/mongodb/mongodb-cli).

> This branch is no longer maintained and will be removed soon. Please update your references accordingly.

Reviewer: ```suggestion
This branch is no longer maintained and is read-only now Please update your references accordingly.
```

# MongoDB CLI (Deprecated)

![GO tests](https://github.com/mongodb/mongodb-cli/workflows/GO%20tests/badge.svg)

> ![golangci-lint](https://github.com/mongodb/mongodb-cli/workflows/golangci-lint/badge.svg)

Reviewer 2: as it's deprecated, what do you think about removing the badges ?

Author: done :)
